chatterbot/conversation/statement.py

```diff
@@ -19,6 +19,9 @@ def __init__(self, text, **kwargs):
         self.text = text
         self.in_response_to = kwargs.pop('in_response_to', [])
 
+        # Store tags information into statements
+        self.tags = kwargs.pop('tags', '')
```

Comment:
It should be string because I am iterating each category and update statement and tag information according, The actual code located [here ](https://github.com/gunthercox/ChatterBot/pull/934/commits/17ffd2938521396c51ed703bbb9af90512213572#diff-7199425a317c16340c90c466ad49584bR123)

``` Python
 for category in corpora.categories:
```